osp/core/utils/schema_validation.py

```diff
@@ -97,9 +98,24 @@ def _load_data_model_from_yaml(data_model_file):
 
 def _check_cuds_object_cardinality(origin_cuds, dest_oclass, rel, constraints):
 
-    actual_cardinality = len(
-        origin_cuds.get(rel=get_entity(rel), oclass=get_entity(dest_oclass))
-    )
+    rel_entity = get_entity(rel)
+
+    if type(rel_entity) == OntologyRelationship:
+        actual_cardinality = len(
+            origin_cuds.get(rel=rel_entity, oclass=get_entity(dest_oclass))
+        )
+    elif type(rel_entity) == OntologyAttribute:
+        # No datatype checking since this is already done when Cuds are
+        # instanciated or imported from a file
```

Comment:
```suggestion
        # instantiated or imported from a file
```

Please apply this suggestion (**required**).